include/libopencm3/efm32/hg/usb.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2015 Kuldeep Singh Dhaka <kuldeepdhaka9@gmail.com>
```

Comment:
I don't see how kuldeep had anything to do with _this_ file.  this is all efm32 usb, which the lg didn't have?